DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -95,12 +99,35 @@ public class DcAccounts {
                 let dcContext = get(id: accountId)
                 dcContext.setConfig("verified_one_on_one_chats", "1")
             }
+
+            rpcPointer = dc_jsonrpc_init(accountsPointer)
         }
     }
 
     public func closeDatabase() {
-        dc_accounts_unref(accountsPointer)
-        accountsPointer = nil
+        if rpcPointer != nil {
+            dc_jsonrpc_unref(rpcPointer)
+            rpcPointer = nil
+        }
+
+        if accountsPointer != nil {
+            dc_accounts_unref(accountsPointer)
+            accountsPointer = nil
+        }
+    }
+
+    @discardableResult
+    public func blockingCall(method: String, params: [AnyObject]) -> Data? {
```

Comment:
Is there a reason to use `JSONSerialization` and `AnyObject` instead of `Codable` and `JSONEncoder`?

Reply:
is the one considered better as the other?

`JSONEncoder` did not work for me at the first place, maybe i did sth. wrong. however, as `JSONSerialization` works out with very few code as well, i also did not investigate further as i found a solution.

Reply:
i just double-checked: `JSONEncoder` cannot encode `[AnyObject]` as not derived from `Codeable`; it would require some additional recursive wrapper. as we're already using `JSONSerialization` in other parts, it is probably fine to stay with that for now; tweaking things here seems not worth the effort currently, we can iterate over that at any point if that becomes an issue